### DISCUSSION TITLE
Documentation for oAuth Provider Icons ?

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9668] [Archive]
I know there are some icons available. Sadly no documentation is there. Is it also possible to add an own svg icon ?

**Comment by rijkvanzanten** [comment]
It's all [Material Icons](https://fonts.google.com/icons) and the following logos:

https://github.com/directus/directus/blob/ca19cb1a999a030ee69703eda26dea809014665b/app/src/components/v-icon/v-icon.vue#L66-L523.

For security reasons, it's not currently possible to use a custom SVG

**Reply by NilsBaumgartner1994** [reply]
Would be nice if that is added to the docs

**Reply by azrikahar** [reply]
In the meantime, just to elaborate on the above social icons, they are the brands variant of Font Awesome 5: https://fontawesome.com/v5.15/icons?d=gallery&s=brands. Hope that's clearer & easier to search.

**Reply by NilsBaumgartner1994** [reply]
Yeah that helps, but as for my purpose like adding an auth possibility for my university Osnabrück this is semi helpfull.
